src/Robo/Commands/Blt/UpdateCommand.php

```diff
@@ -351,26 +342,6 @@ protected function convertWindowsPathToCygwinPath($path) {
     return str_replace('\\', '/', preg_replace('/([A-Z]):/i', '/cygdrive/$1', $path));
   }
 
-  /**
-   * Updates project BLT .yml files with new key value pairs from upstream.
-   *
-   * This WILL NOT overwrite existing values.
-   */
-  protected function mungeProjectYml() {
```

Comment:
I removed this function because I can't imagine why a project would have a pre-existing blt.yml file that contains customizations that need to be preserved.